components/CovidMap.js

```diff
@@ -106,7 +106,23 @@ function formatTick(ms) {
   return moment(new Date(ms)).format('MMM DD');
 }
 
-function renderDateTime(date) {
+function renderDateTime(minSliderDate, date) {
```

Comment:
What do you think about reducing some of the duplicated JSX in this function? Here's some completely untested code to illustrate about what I'm thinking of.

```javascript
function renderDateTime(minSliderDate, date) {
  let formattedDate;

  if (minSliderDate.getTime() === date.getTime()) {
    formattedDate = moment(date).format('MMM DD');
  } else {
    const formattedStartDate = moment(minSliderDate).format('MMM DD');
    const formattedEndDate = moment(date).format('MMM DD');
    formattedDate = `${formattedStartDate} &ndash; ${formattedEndDate}`;
  }

  return (
    <div
      style={{
        width: '100%',
        textAlign: 'center',
        fontFamily: 'Arial',
        margin: 5,
      }}
    >
      <div style={{ fontSize: 12 }}>
        <b>{formattedDate}</b>
      </div>
    </div>
  )
}
```

Reply:
It worked, I just had to change &ndash; to \u2013